### ISSUE TITLE
MDL crashes when using -j option with configuration file

### ISSUE THREAD

**Issue by PowerSchill** [markdownlint/markdownlint#285]
## Description

When using the `-j` option to display the output in JSON format with a configuration file present generates an error.

### Environment

* macOS Catalina (10.15.1)
* homebrew with rbenv
* ruby version 2.6.5

**MDL Version**

0.8.0

### Expected Behavior

normal output indicating syntax issues

### Actual Behavior

Trace below is generated. 

```
Traceback (most recent call last):
	3: from /Users/user/.rbenv/versions/2.6.5/bin/mdl:23:in `<main>'
	2: from /Users/user/.rbenv/versions/2.6.5/bin/mdl:23:in `load'
	1: from /Users/user/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/mdl-0.8.0/bin/mdl:10:in `<top (required)>'
/Users/user/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/mdl-0.8.0/lib/mdl.rb:109:in `run': uninitialized constant MarkdownLint::JSON (NameError)
```
## Replication Case

1. Execute `mdl -j .` on a folder with markdown files and you should get normal output. 
2. Create a `.mdlrc` file in the project with the following content:

```bash
# Only check rules for files known to git
git_recurce true

# Ignore line length
# Ignore multiple headings with the same content
rules "~MD013", "~MD024"
```
3. Execute `mdl -j .` on a folder with markdown files and you will get stacktrace.

**Comment by jaymzh** [comment]
This is fixed in master. Will close when next version is released.

**Comment by jaymzh** [comment]
Fixed in 0.9.0.
